datetime_1.py

```diff
@@ -0,0 +1,164 @@
+from datetime import datetime
+import os
+import time
+
+
+output_number =\
+{0:\
+    """
+    ■■■■■
```

Comment:
I would pre-split digits like this
[
[xxx],
[x x]
]
in what case you wouldn't  have need to split them every time below